lambdas/ingester/src/templates/about-page.jsx

```diff
@@ -0,0 +1,113 @@
+const { h } = require('hyperons');
+const { GraphvizLibrary } = require('./components/structure');
+
+const About = () => {
+	return (
+		<main className="o-layout__main">
+			<GraphvizLibrary />
+			<div className="o-layout-typography">
+				<h1 className="group-title">
+					About RUNBOOK.MD (Github markdown importer)
+				</h1>
+				<p>
+					This project is an attempt to provide tech teams at the FT
+					with a simple method of populating Biz Ops from the content
+					of any RUNBOOK.MD files in their GitHub repository. It
+					comprises of these parts: A <a href="#form">Form</a>, A
+					Github <a href="#webhook">Webhook</a> and
+					<a href="#changeapi">Change API Kinesis Events</a>.
+				</p>
+				<div id="flow" />
+				<p />
+				<h3 id="form">The Form.</h3>A form is provided{' '}
+				<a href="https:/biz-ops.in.ft.com/runbook.md">here</a> for users
+				who wish to familiarise themselves with the RUNBOOK.MD concept.
+				It provides sample RUNBOOK.MD content and invites the user to
+				enter markdown text. On submission the text is then run through
+				the <a href="#parser">parser</a>, the{' '}
+				<a href="#validator">validator</a> and the{' '}
+				<a href="#scoring">scorer</a>. Any errors are then{' '}
+				<a href="#displayed">displayed.</a>. If the form validation is
+				successful then the user may, on entry of a systemCode and a Biz
+				Ops apikey, trigger the validated details to be written to Biz
+				Ops.
+				<p />
+				<h3 id="webhook">The GitHub Webhook</h3>
+				Some text here about the webhook.
+				<p />
+				<h3 id="changeapi">Change API Kinesis Events</h3>
```

Comment:
```suggestion
				<h3 id="changeapi">Change API Events</h3>
```

Because kinesis is an implementation detail no-one needs to know about